hax-lib/src/proc_macros.rs

```diff
@@ -2,9 +2,9 @@
 //! proc-macro crate cannot export anything but procedural macros.
 
 pub use hax_lib_macros::{
-    attributes, decreases, ensures, exclude, impl_fn_decoration, include, lemma, loop_decreases,
-    loop_invariant, opaque, opaque_type, refinement_type, requires, trait_fn_decoration,
-    transparent,
+    attributes, decreases, ensures, exclude, impl_fn_decoration, include, lean_proof, lemma,
```

Comment:
Let's not expose `lean_proof` in the root of that crate, exporting it as `lean::proof` is good